linux/src/sync_linux.c

```diff
@@ -53,6 +52,7 @@ IMPLEMENT_MOCKABLE_FUNCTION(, WAIT_ON_ADDRESS_RESULT, wait_on_address, volatile_
         }
         else
         {
+            LogError("failure in syscall, errno=%d (%s)", errno, strerror(errno));

```

Comment:
>strerror [](http://example.com/codeflow?start=65&length=8)

we should decide on one or the other (strerror/strerror_r) functions to use.

errno is always threadsafe. strerror... "maybe".

Reply:
Also errno needs to be reset before the call and observed after the call. Or otherwise you can inherit some "old" errno.

This is sadly here and everywhere